biz.aQute.bndlib/src/aQute/bnd/exporter/runbundles/RunbundlesExporter.java

```diff
@@ -33,23 +36,35 @@ public RunbundlesExporter() {}
 	public Entry<String, Resource> export(String type, Project project, Map<String, String> options) throws Exception {
 		project.prepare();
 		Collection<Container> runbundles = project.getRunbundles();
-		Jar jar = new Jar(project.getName());
+		String prefix = options.getOrDefault(TARGET_DIR, "");
+		if (!prefix.endsWith("/")) {
```

Comment:
This means the default for `prefix` will be `/` since you default to empty string which does not end with slash and that is then suffixed with `/`. I don't think that is the proper default. Empty string should be the proper default.
```suggestion
		if (!prefix.isEmpty() && !prefix.endsWith("/")) {
```

Reply:
done